recipes/libverto/all/conanfile.py

```diff
@@ -106,62 +112,60 @@ def requirements(self):
-            self.build_requires("msys2/cci.latest")
-
-    @contextlib.contextmanager
-    def _build_context(self):
-        if self.settings.compiler == "Visual Studio":
-            with tools.vcvars(self):
-                env = {
-                    "CC": "{} cl -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
-                    "CXX": "{} cl -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
-                    "LD": "{} link -nologo".format(tools.unix_path(self.deps_user_info["automake"].compile)),
-                    "AR": "{} lib".format(tools.unix_path(self.deps_user_info["automake"].ar_lib)),
-                }
-                with tools.environment_append(env):
-                    yield
-        else:
-            yield
-
-    def _configure_autotools(self):
-        if self._autotools:
-            return self._autotools
-        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
-        self._autotools.libs = []
+        self.build_requires("pkgconf/1.9.3")
+        self.build_requires("libtool/2.4.7")
+        if self._settings_build.os == "Windows":
+            self.win_bash = True
+            if not self.conf.get("tools.microsoft.bash:path", check_type=str):
+                self.tool_requires("msys2/cci.latest")
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
+        tc = AutotoolsToolchain(self)
         yes_no = lambda v: "yes" if v else "no"
         yes_no_builtin = lambda v: {"external": "yes", "False": "no", "builtin": "builtin"}[str(v)]
-        args = [
-            "--enable-shared={}".format(yes_no(self.options.shared)),
-            "--enable-static={}".format(yes_no(not self.options.shared)),
-            "--with-pthread={}".format(yes_no(self.options.get_safe("pthread", False))),
-            "--with-glib={}".format(yes_no_builtin(self.options.with_glib)),
-            "--with-libev={}".format(yes_no_builtin(self.options.with_libev)),
-            "--with-libevent={}".format(yes_no_builtin(self.options.with_libevent)),
-            "--with-tevent={}".format(yes_no_builtin(self.options.with_tevent)),
-        ]
-        self._autotools.configure(args=args, configure_dir=self._source_subfolder)
-        return self._autotools
+        tc.configure_args.extend([
+            f"--with-pthread={yes_no(self.options.get_safe('pthread'))}",
+            f"--with-glib={yes_no_builtin(self.options.with_glib)}",
+            f"--with-libev={yes_no_builtin(self.options.with_libev)}",
+            f"--with-libevent={yes_no_builtin(self.options.with_libevent)}",
+            f"--with-tevent={yes_no_builtin(self.options.with_tevent)}",
+            ])
+
+        env = tc.environment()
+        # if is_msvc(self):
+        #     # FIXME: Use the conf once https://github.com/conan-io/conan-center-index/pull/12898 is merged
```

Comment:
The recipe behind this PR was updated.. not sure if the comments make sense any more

Reply:
Looks like we are really waiting for #15023

Reply:
> Looks like we are really waiting for #15023

No, automake has been migrated, you shouldn't wait this PR.

Reply:
this recipe raises for msvc anyway, so I don't understand these comments.